Compiler warning: tautological comparison

During make -j
```
~/projects/visualboyadvance-m/src/gba/ereader.cpp:459:23: warning: comparison of unsigned expression >= 0 is always true [-Wtautological-compare]
        if ((reg[0].I >= 0) && (reg[0].I <= 0x10)) {
             ~~~~~~~~ ^  ~
1 warning generated.
```
This might be causing another issue.

--- a/src/gba/ereader.cpp
+++ b/src/gba/ereader.cpp
@@ -456,7 +456,7 @@ void BIOS_EReader_ScanCard(int swi_num)
             THUMB_PREFETCH
     } else if ((swi_num == 0xE3) || (swi_num == 0xE5)) //Dotcode data
     {
-        if ((reg[0].I >= 0) && (reg[0].I <= 0x10)) {
+        if (reg[0].I <= 0x10) {
             if (decodestate == 0) {
                 for (i = 0x17; i >= 0; i--) {
                     if ((0x17 - i) < 8)

